.github/workflows/release.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           java-version: '11'
 
       - name: setup Clojure
-        uses: DeLaGuardo/setup-clojure@2.0
+        uses: DeLaGuardo/setup-clojure@DeLaGuardo/setup-clojure
```

Comment:
Should this be `e73bf2b6435244b2c9c5c226ae5022d91d0ce702`?

Reply:
of course it should, I'm a dumb. and nice catch!

Reply:
Fixed